### PR TITLE
Check again if we need to use cork after initializing parsers

### PR DESCRIPTION
**Pull request by hadrielk**
Fix for issue #1851.

### CODE REVIEW

File: Units/mtable-simple-with-table-extending.d/args.ctags [hunk 1]
```diff
@@ -6,6 +6,8 @@
 --kinddef-X=v,var,variables
 --kinddef-X=f,fun,functions
 
+--kinds-X=+{var}
+
```
Comment (masatake) [inline Units/mtable-simple-with-table-extending.d/args.ctags]: Can I ask you to make a separate test case?
This test case will be important when I rewrite the code for parser initialization.
I think this should go to Tmain.

Comment (hadrielk) [reply Units/mtable-simple-with-table-extending.d/args.ctags]: Yes I'll make it a separate test case and update the PR.

**Comment by masatake** [comment]
@hadrielk, could you look at #1858 ?

**Comment by masatake** [comment]
#1858 closes the original issue.
@hadrielk, thank you for taking your time for reporting the original issue and this pull request.

**Pull request closed without merging**